querybook/webapp/components/StatementResultTable/StatementResultTable.tsx

```diff
@@ -186,10 +181,11 @@ export const StatementResultTable = React.forwardRef<
                 cols={columns}
                 showPagination={showPagination}
                 formatCell={(index, column, row) => {
+                    const value = row[index];
                     const transformer = getTransformerForColumn(index);
-                    return transformer
-                        ? transformer.transform(row[index])
-                        : row[index];
+                    return transformer && !isCellValNull(value)
```

Comment:
can we return early when `isCellValNull(value)` is true, to save the calling for `getTransformerForColumn`